frontend/src/app/main/ui/workspace/viewport/widgets.cljs

```diff
@@ -0,0 +1,183 @@
+; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020-2021 UXBOX Labs SL
+
+(ns app.main.ui.workspace.viewport.widgets
+  (:require
+   [app.common.geom.point :as gpt]
+   [app.common.geom.shapes :as gsh]
+   [app.main.refs :as refs]
+   [app.main.streams :as ms]
+   [app.main.ui.hooks :as hooks]
+   [app.main.ui.workspace.shapes.outline :refer [outline]]
+   [app.main.ui.workspace.shapes.path.actions :refer [path-actions]]
+   [clojure.set :as set]
+   [rumext.alpha :as mf]
+   [app.common.pages :as cp]
+   [app.util.dom :as dom]
+   ))
+
+(mf/defc coordinates
```

Comment:
Duplicate of `app.main.ui.workspace.coordinates/coordinates`.

Reply:
I'll remove the one in widgets.